meson.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
do we actually need a meson.sh?  I think an update to the README.md should be sufficient for the "commonly used args".

Reply:
I say make this simple: if you want to switch to meson then remove the autotools stuff at the same time. Or else there will be no reason to switch it for distributions.